plugins/workbench/src/components/internal/Application.svelte

```diff
@@ -93,6 +93,7 @@
     if (str === 'Pages') return 'New page'
     if (str === 'Tasks') return 'New task'
     if (str === 'Vacancies') return 'Add candidate'
+    if (str === 'Calendar') return 'Add event'
```

Comment:
@ArtyomSavchenko Please rebase, this is not required anymore and you need to use new Create extension point.